<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>retencion</class>
 <widget class="QDialog" name="retencion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de retenciones IRPF</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Cuenta retención:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cta_retlineEdit">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="cta_retlabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>335</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>- -</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Cuenta Retenido:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buscaretenidopushButton">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="retenidolineEdit">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ctaretenidolabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>335</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>- -</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="arrendamientocheckBox">
     <property name="text">
      <string>Rendimientos procedentes del arrendamiento de inmuebles Urbanos</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Clave percepciones:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="clavecomboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>495</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Base percepciones:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="baselineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tipo retención (%):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tiporetlineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="dinerariosgroupBox">
       <property name="title">
        <string>Rendimientos dinerarios</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Retención:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="retlineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="especiegroupBox">
       <property name="title">
        <string>Rendimientos en especie</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Ingresos a cuenta efectuados</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ing_a_ctalineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Ingresos a cuenta repercutidos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ing_a_cta_replineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Datos a suministrar si se utilizan cuentas genéricas de retenidos</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nombrelineEdit">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>CIF:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ciflineEdit">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Provincia:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="provinciacomboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>retencion</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
